flexget/components/seen/cli.py

```diff
@@ -57,8 +79,11 @@ def seen_search(options, session=None):
         else:
             console("Could not parse IMDB ID")
     else:
-        search_term = '%' + options.search_term + '%'
-    seen_entries = db.search(value=search_term, status=None, session=session)
+        search_term = options.search_term.replace("*", "%").replace("?", "_")
```

Comment:
If we are switching the wildcard, should it automatically escape % and _ from the search so they end up searching for the literal characters?